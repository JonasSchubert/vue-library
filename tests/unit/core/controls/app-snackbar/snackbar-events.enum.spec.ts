import { SnackbarEvents } from '@/core/controls/app-snackbar/snackbar-events.enum';

describe('SnackbarEvents', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(SnackbarEvents).toMatchSnapshot();
  });
});
