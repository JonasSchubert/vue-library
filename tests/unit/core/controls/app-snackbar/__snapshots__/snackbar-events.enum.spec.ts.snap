// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnackbarEvents should not change unexpected 1`] = `
Object {
  "hide": "snackbar.hide",
  "show": "snackbar.show",
}
`;
